web/portal/user/src/App.tsx

```diff
@@ -36,7 +48,20 @@ export default function App(): JSX.Element {
     apiSpecStore.init();
     authStore.setSessionStoragePrefix('IP-user-');
     authStore.init();
-  }, [apiSpecStore, authStore, token, setLanguages, setLoginProps]);
+
+    userStatusResetProfile();
+    userStatusInit();
+    loadStatus();
+  }, [
+    apiSpecStore,
+    authStore,
+    token,
+    setLanguages,
+    setLoginProps,
+    userStatusResetProfile,
+    userStatusInit,
+    loadStatus,
```

Comment:
I don't get why you need these new effect triggers

Reply:
The linter complaints if those are missing:
`React Hook useEffect has missing dependencies: 'loadStatus' and 'userStatusInit'. Either include them or remove the dependency array`